lib/build.js

```diff
@@ -8,7 +8,16 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
-	const dockerfile = `FROM node:${context.version}-onbuild`;
+	const dockerfile = [
+		`FROM node:${context.version}`,
+		'RUN mkdir -p /usr/src/app',
```

Comment:
I don't think it's necessary, since you have `WORKDIR` below already.

Reply:
Yep, you're right. I didn't realize that setting `WORKDIR` creates the path automatically. Thanks for the tip!  I've removed this line of code.